README.md

```diff
@@ -86,6 +87,41 @@ As a modern Python-based environment, GamestonkTerminal opens access to numerous
 
 ## Getting Started
 
+### Docker Installation - *new and improved*
+
+0. First step is to make sure docker desktop is installed.  Install links can be found [here](https://www.docker.com/products/docker-desktop).  To confirm that your docker desktop is downloaded and running, open a command prompt or terminal and enter
+`docker info`.  If you get the following you are not running the docker desktop:
+```bash
+Server:
+ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
+```
+Open the docker desktop app in this case.
+
+1. Download the latest docker image.
+
+```bash
+docker pull ghcr.io/gamestonkterminal/gst-poetry:latest
+```
+
+Upon running this the first time, you should see the various layers downloading (note the random series of letters numbers will vary).
+![Screen Shot 2021-09-08 at 10 41 08 AM](https://user-images.githubusercontent.com/18151143/132531075-7d7f7e71-4fcb-435c-9bb3-466d7077eba4.png)
+
+Once the download is complete, confirm that the image has been created by doing `docker images`.  You should see something similar to
+```bash
+REPOSITORY                             TAG       IMAGE ID       CREATED        SIZE
+ghcr.io/gamestonkterminal/gst-poetry   latest    e2bbeebcc73c   42 hours ago   2.02GB
+```
+
+2. Run a container
+
+You are now ready to run the terminal.
+
+```docker run -it --rm ghcr.io/gamestonkterminal/gst-poetry:latest```
+
+This will open up the terminal in your command prompt or terminal.  Note that this has provided now environment file, so you will not be able to view plots or use keys at this stage.
+
+To read more on adding the environment keys and how to configure your X-server to show plots, hop over to the [Advanced Docker Setup](/DOCKER_ADVANCED.md).
+
 ### Install
```

Comment:
I would also update this header to something like - Install with local Python, Git and Python dependencies